<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchoolDetailsWindow</class>
 <widget class="QMainWindow" name="SchoolDetailsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>School Details</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/assets/icons/logo.png</normaloff>:/icons/assets/icons/logo.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#txt_business_logo {
	border-radius: 5px;
	background-color: #fff;
	padding: 5px;
}

#label {
	background-color: #81d4fa;
	color: #fff;
}

#label_5 {
	color: #fff;
}

#btn_save {
	background-color: #29b6f6;
	border-radius: 5px;
	padding: 5px 20px;
	color: white;
}

#btn_clear {
	background-color: #80d8ff;
	border-radius: 5px;
	padding: 5px 20px;
	color: white;
}

#btn_cancel {
	background-color: #b0bec5;
	border-radius: 5px;
	padding: 5px 20px;
	color: white;
}

QLineEdit {
	border-radius: 5px;
	padding: 5px 5px;
}

QDateEdit {
	border-radius: 5px;
	padding: 5px 5px;
}

QComboBox {
	border-radius: 5px;
	padding: 5px 5px;
}

QTextEdit {
	border-radius: 10px;
	padding: 0px 0px;
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignTop">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Calibri</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>School Details</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignVCenter">
     <widget class="QWidget" name="main_details_widget" native="true">
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_18">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="business_logo_frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_17">
          <item>
           <widget class="QLabel" name="txt_business_logo">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>120</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Logo</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="input_field_frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_16">
          <property name="spacing">
           <number>10</number>
          </property>
          <item alignment="Qt::AlignRight">
           <widget class="QLineEdit" name="txt_business_name">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLineEdit" name="txt_business_contact">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QLineEdit" name="txt_business_address">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
             </font>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="labels_frame">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_15">
          <property name="spacing">
           <number>20</number>
          </property>
          <item>
           <widget class="QLabel" name="b_name_lbl">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>School Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="b_contact_lbl">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Contact</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="b_address_lbl">
            <property name="font">
             <font>
              <family>Calibri</family>
              <pointsize>14</pointsize>
              <weight>50</weight>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Address</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="bottom_frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_cancel">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/assets/icons/close.png</normaloff>:/icons/assets/icons/close.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clear">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/assets/icons/clear.png</normaloff>:/icons/assets/icons/clear.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_save">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/icons/assets/icons/save.png</normaloff>:/icons/assets/icons/save.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
